pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,76 @@
+Your full node needs a backup plan to replay the history of the network in case it falls out of sync. You can back up your node in one of two ways:
 
-### State sync (Alternative to snapshots)
+1. Using a **snapshot**, which contains a compressed copy of the application state at a point in time, saved on the system that your node runs on. If your node falls out of sync, the snapshot allows the node to quickly recover to the application state when your snapshot was taken. This speeds up the syncing process significantly. To use a snapshot to back up your full node, install a snapshot from a [snapshot service](../infrastructure_providers-network/resources.mdx#snapshot-service).
 
-State Sync enables a new node to join the network by obtaining a snapshot of the application state from a state sync node at a recent height. This eliminates the need to fetch and replay all historical blocks. A list of state sync services with instructions are listed below
+2. Using **state sync**, which enables a your node to obtain a snapshot of the application state from a state sync node. This speeds up the syncing process significantly. To use state sync to back up your full node, follow the instructions from a state sync service from the list below:
 
-https://polkachu.com/state_sync/dydx
+    - https://polkachu.com/state_sync/dydx
+    - https://services.lavenderfive.com/mainnet/dydx/statesync
+    - https://autostake.com/networks/dydx/#state-sync
 
-https://services.lavenderfive.com/mainnet/dydx/statesync
+<!-- get into the differences between this. "This speeds up the syncing process significantly" -->
 
-https://autostake.com/networks/dydx/#state-sync
-
-### Address Book
-
-The **`addrbook.json`** file is used to store configuration details that help a node connect to other peers in the network more efficiently. This can be obtained from the one of the below services and needs to be stored in the `config` folder.
-
-https://polkachu.com/addrbooks/dydx
-
-https://services.lavenderfive.com/mainnet/dydx#latest-addrbook
-
-https://autostake.com/networks/dydx/
-
-## Pruning Settings
-
-For optimal storage, use the following pruning setting in the app.toml file:
+## Optimize Pruning Settings
+In general, dYdX recommends the following pruning setting, configured in your `app.toml` file:
 
 ```bash
-# 2 latest states will be kept; pruning at 10 block intervals.
-pruning = "everything"
+# app.toml
+pruning = "everything" # 2 latest states will be kept; pruning at 10 block intervals
 ```
 
-If the full node is being used for historical queries, a custom strategy should be used to maintain more states. This will increase storage requirements.
+However, if you want to use your node to query historical data, configure a custom pruning strategy to retain more states. Retaining more states increases storage requirements.
+
+## Next Steps
+When you've configured your node...
+Learn more on the [next]() page.
```

Comment:
what should the next steps be? I imagine there are a few directions a user could go from here, so I think we should list the most common few

Maybe:
- stream chain data
- create your own orderbook
- create your own historical price chart

Reply:
would be good to research and figure out what the best next step would be (ask Affan if needed)